source/default/_patterns/02-molecules/carousel/_carousel.twig

```diff
@@ -51,3 +50,5 @@
     {% endblock controls %}
   </div>
 </div>
+
+
```

Comment:
Extra newline if you feel like removing.

Reply:
Good catch!